client/verta/verta/endpoint/_kafka_settings.py

```diff
@@ -10,6 +10,12 @@ class KafkaSettings(object):
     For use during :class:`~verta.endpoint.Endpoint` creation or with
     :meth:`Endpoint.update() <verta.endpoint.Endpoint.update>`.
 
+    Search for available Kafka topics with :meth:`Client.get_kafka_topics() <verta.Client.get_kafka_topics>`.
+
+    .. versionadded:: 0.22.3
+        ``cluster_config_id`` is now fetched from the active Kafka
+        configuration by default.  It remains optional for backwards
+        compatibility.
```

Comment:
I think we can leave this out.

```suggestion
        configuration if not provided, if there is only one configured.
```

It's not optional for backwards compatibility; like we mentioned in Slack, it's for the future possibility of multiple Kafka clusters. Either way, I don't think we need to justify it in the docstring.

Reply:
Fixed